src/templates/LectureView/VideoPlayer.js

```diff
@@ -1,12 +1,22 @@
-import React, { useContext } from 'react'
+import React, { useContext, useState } from 'react'
 import ReactPlayer from 'react-player'
 import PropTypes from 'prop-types'
 import { AuthContext } from '../../contexts/AuthContext'
 import { addProfile } from '../../services/api'
+import CourseRating from '../../components/Courses/CourseRating'
 
-const VideoPlayer = ({ url, lectureStrapiId, courseStrapiId }) => {
+const VideoPlayer = ({
+  url,
+  lectureStrapiId,
+  courseStrapiId,
+  lecturesLength,
+  position,
+}) => {
   const DEFENITION_OF_COMPLETED = 0.8
   const { currentUser, isLoggedIn, setCurrentUser } = useContext(AuthContext)
+  const showFeedback = (position * 10) % lecturesLength === 0
```

Comment:
why `*10` - are we adding positions to the DB like 10,20 etc...

Reply:
As I said I didn't know what to put. This variable is for displaying the feedback form.
I added the 10 to increase the possibility of showing the form.